protobuf/syft_proto/execution/v1/communication_action.proto

```diff
@@ -3,13 +3,22 @@ syntax = "proto3";
 package syft_proto.execution.v1;
 option java_package = "org.openmined.syftproto.execution.v1";
 
+import "syft_proto/execution/v1/placeholder_id.proto";
+import "syft_proto/generic/pointers/v1/pointer_tensor.proto";
 import "syft_proto/types/syft/v1/arg.proto";
 import "syft_proto/types/syft/v1/id.proto";
+import "syft_proto/types/torch/v1/tensor.proto";
 
 message CommunicationAction {
-    string name = 1;
-    syft_proto.types.syft.v1.Id obj_id = 2;
-    syft_proto.types.syft.v1.Id source = 3;
-    repeated syft_proto.types.syft.v1.Id destinations = 4;
-    map<string, syft_proto.types.syft.v1.Arg> kwargs = 5;
+    string command = 1;
+    oneof target {
```

Comment:
I haven't look at all what you did yet so not sure it's equivalent, but do you have any thought about having a target_id instead of the whole object on which the action is performed here?

Reply:
I'm fine with it as long we can make it consistent across the two `Action` types.

Reply:
I think ids would make things a bit easier (but I'm not sure I see everything, that's why I asked you opinion)
Anyway, I'm fine with what we have now, it can be changed later if really needed